index.rst

```diff
@@ -1,57 +1,343 @@
+Shuffling the order of queued scripts is also possible.
+Lastly, users can quickly relaunch previously run scripts without having to re-enter any modifications to the default configuration.
 
-  Sections
-  ========
+.. figure:: /_static/ScriptQueueUI.png
+   :name: fig-scriptqueueui
+   :target: ../_images/ScriptQueueUI.png
+   :alt: LOVE ScriptQueue user interface
 
-  and
+   A screenshot of the LOVE interface to interact with the ScriptQueue.
 
-  Subsections
-  -----------
+In general, scripts require minimum interaction to be executed except, of course, for the occasional configuration.
+A database of configurations will be available for users to execute the most common set of operations.
+Users also have the possibility to edit configurations from the user interface, which also provides on-the-fly schema validation.
 
-  and
+In general, using the ScriptQueue requires some familiarity with the observatory system and minimal set of software development skills.
+Users should be able to, after inspecting the LOVE status screens, determine the state of the observatory and its readiness to perform certain types of operations.
+Other than that, some knowledge about `yaml`_ and `json schema`_ may be useful for writing and inspecting script configurations, though :ref:`LOVE <fig-scriptqueueui>` will eventually provide features to help understanding and validating them prior to execution.
 
-  Subsubsections
-  ^^^^^^^^^^^^^^
+.. _yaml: https://yaml.org/spec/1.2/spec.html
+.. _json schema: http://json-schema.org
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
+.. _SAL_Scripts:
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
+SAL Scripts
+-----------
 
-     Caption text.
+`SAL Scripts`_ are the files which contain the logic and coordination of events and CSCs that get executed by the `ScriptQueue`_.
+It is not expected these will be modified during standard night-time operations.
+All `SAL Scripts`_ vary greatly in complexity of the operation being performed.
+For example, a SAL script that performs a relatively fundamental task is `Enable MTCS <https://github.com/lsst-ts/ts_standardscripts/blob/develop/python/lsst/ts/standardscripts/maintel/enable_mtcs.py>`_ which brings all components of the TCS to the enabled state.
+The script is normally a launched using the default configuration which enables all components, however, the flexibility is present to only enable a subset if the observer chooses to do so.
+A more complicated script, such as `Prepare for On-Sky <https://github.com/lsst-ts/ts_standardscripts/blob/develop/python/lsst/ts/standardscripts/auxtel/prepare_for_onsky.py>`_ performs a series of order-specific operations to bring the systems online, then open the dome and telescope safely.
+`SAL Scripts`_ obey strict formatting requirements and must contain specific information in order to be capable of execution.
```

Comment:
New paragraph? Also, I think this phrase could be removed altogether. I don't think it is adding any relevant information.

Reply:
I think it's important as most people see a script as just a series of commands in a blank file... like a bash script.
I did make it a new paragraph